web/satellite/src/utils/constants/analyticsEventNames.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+export const EVENTS = {
```

Comment:
will it be better if we create smth like
enum SegmentEvent {
    CLICKED_LOGO = 'clicked-on-logo',
    CLICKED_LOGIN = 'clicked-on-login',
    CLICKED_BACK_TO_LOGIN = 'clicked-on-back-to-login',
}